### PR TITLE
Removing Duplications of some DD Algos

### PR DESCRIPTION
**Pull request by boudoul**
DD Algos were duplicated after the PR #7815 
The ones duplicated for 2017 have then been removed from SLHCUpgradeSimulations/Geometry (the Geometry/TrackerCommonData is more adequate) 

@rovere @VinInn @atricomi @venturia

### CODE REVIEW

**Comment by ianna** [comment]
+1

File: Geometry/TrackerCommonData/plugins/DDPixBarLayerUpgradeAlgo.cc [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #include "DetectorDescription/Core/interface/DDMaterial.h"
 #include "DetectorDescription/Core/interface/DDCurrentNamespace.h"
 #include "DetectorDescription/Core/interface/DDSplit.h"
-#include "SLHCUpgradeSimulations/Geometry/interface/DDPixBarLayerUpgradeAlgo.h"
+#include "Geometry/TrackerCommonData/plugins/DDPixBarLayerUpgradeAlgo.h"
```
Comment (ktf) [inline Geometry/TrackerCommonData/plugins/DDPixBarLayerUpgradeAlgo.cc]: Do we actually allow including across plugins?

Comment (ktf) [inline Geometry/TrackerCommonData/plugins/DDPixBarLayerUpgradeAlgo.cc]: Nevermind...

**Comment by davidlange6** [comment]
+1

**Pull request merged**